.prettierrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('kcd-scripts/prettier')
```

Comment:
Actually, I'm a big fan of prettier w/ no config.  I like all of the justifications they make for their defaults.  can we just remove this file please